enos/enos-scenario-autopilot.hcl

```diff
@@ -632,12 +632,50 @@ scenario "autopilot" {
     }
   }
 
+  step "verify_removed" {
+    description = <<-EOF
+      Verify that the removed nodes are marked as such
+    EOF
+    module      = semverconstraint(var.vault_upgrade_initial_version, ">=1.19.0-0") ? "vault_verify_removed_node" : "vault_verify_removed_node_shim"
+    depends_on = [
+      step.create_vault_cluster,
+      step.create_vault_cluster_targets,
+      step.get_updated_vault_cluster_ips,
+      step.raft_remove_peer,
+    ]
+
+    providers = {
+      enos = local.enos_provider[matrix.distro]
+    }
+
+    verifies = [
+      quality.vault_raft_removed_after_restart,
```

Comment:
can we run `enos fmt` to fix the formatting here? (`make fmt` in the `enos dir ought to do this)